Cargo.toml

```
@@ -21,7 +21,7 @@ inventory = { version = "0.1.4", optional = true }
 libc = "0.2.62"
 parking_lot = "0.11.0"
 num-bigint = { version = "0.4", optional = true }
-num-complex = { version = "0.4", optional = true }
+num-complex = { version = ">= 0.2, <= 0.4", optional = true }
```

as `"0.4"` alone really means `">= 0.4.0, < 0.5.0", I think this needs to be:

```suggestion
num-complex = { version = ">= 0.2, < 0.5", optional = true }
```